tests/unit/featurizers/test_atom.py

```diff
@@ -91,13 +92,16 @@ def test_none(featurizer):
     np.testing.assert_array_equal(featurizer(None), np.zeros(len(featurizer)))
 
 
-def test_atomic_num_bit(atom, x, max_atomic_num):
+def test_atomic_num_bit(atom, x, atomic_num):
     n = atom.GetAtomicNum()
 
-    if n > max_atomic_num:
-        assert x[max_atomic_num] == 1
+    if n == 53: # special check for Iodine
+        assert x[len(atomic_num) - 1] == 1
```

Comment:
I do not think using an iodine example will eliminate these two lines because for atomic number 1-36, it directly corresponds to entries 0 to 35 of the atomic number feature vector. However, iodine has an atomic number of 53 but it is mapped to the 36 index of the feature vector, and therefore need a special check anyway.